applications/luci-app-pbr/Makefile

```diff
@@ -24,6 +24,6 @@ help
 	Version: $(PKG_VERSION)-$(PKG_RELEASE)
 endef
 
-include ../../luci.mk
+include $(TOPDIR)/feeds/luci/luci.mk
```

Comment:
@stangri this change causing that in my local setup (and I'm not even building it):
```
ERROR: please fix package/feeds/applications/luci-app-pbr/Makefile - see logs/package/feeds/applications/luci-app-pbr/dump.txt for details
```
My luci.mk lives in `openwrt/package/feeds/luci.mk` and for all other packages it causes no problem.

Reply:
Thanks for the feedback, could you please provide the content of the `logs/package/feeds/applications/luci-app-pbr/dump.txt`?

Reply:
Here you are:
```
Makefile:27: /home/krant/openwrt/feeds/luci/luci.mk: No such file or directory
make[2]: *** No rule to make target '/home/krant/openwrt/feeds/luci/luci.mk'.  Stop.
```

Reply:
@krant thanks for your prompt reply, I'll revert the change